src/map.cpp

```diff
@@ -4672,6 +4672,28 @@ static void process_vehicle_items( vehicle *cur_veh, int part )
             apply_in_fridge(n);
         }
     }
+
+    const bool washmachine_here = cur_veh->part_flag( part, VPFLAG_WASHING_MACHINE ) && cur_veh->has_part( "WASHING_MACHINE", true );
+    bool washing_machine_finished = false;
+    if( washmachine_here ) {
+        for( auto &n : cur_veh->get_items( part ) ) {
+            const int washing_time = MINUTES( 90 );
+            int time_left = washing_time - calendar::turn.get_turn() + n.bday;
+            static const std::string filthy( "FILTHY" );
+            if( time_left == 0 ) {
```

Comment:
This condition would probably "break" the washing machine if one leaves the reality bubble and returns more than 90 minutes later. `time_left` then becomes negative and provokes infinite washing cycle (along with increasingly negative number of minutes in the message).